Mastermind/Presentation/InputOutput/ConsoleMessages.cs

```diff
@@ -0,0 +1,56 @@
+using Mastermind.Domain.BusinessRules;
+
+namespace Mastermind.Presentation.InputOutput;
+
+public static class ConsoleMessages
+{
+    public const string GetUserGuessMessage = "You can pick your four colours from the following!";
+    public const string GetUserGuessPrompt = "Please enter the numbers to select the respective colours (e.g. 2345) : ";
```

Comment:
Another nitpicky one, but the usual convention is for `Get` to be used as a prefix for functions rather than variables. I would suggest going with something like `UserGuessMessage` instead.